docs/event.md

```diff
@@ -30,6 +30,7 @@ Used to either create or send mock events for use with local webhooks testing.
 | `update-redemption` | Channel Points EventSub event for a redemption being updated.                                              |
 | `raid`              | Channel Raid event with a random viewer count.                                                             |
 | `revoke`            | User authorization revoke event. Uses local Client as set in `twitch configure` or generates one randomly. |
+| `stream_change`     | Stream Changed event.                                                                                      |
```

Comment:
Not your fault, but can we either change this to `streamchange` or `stream-change` to match formatting for the other keywords?

Reply:
Addressed and corrected.